.drone.yml

```diff
@@ -44,15 +44,59 @@ pipeline:
     detach: true
     environment:
       - DOCKER_HOST=tcp://docker:2375
+      - DEBUG=true
+      - OWNCLOUD_APPS_INSTALL=https://github.com/owncloud/testing/releases/download/latest/testing.tar.gz
+      - OWNCLOUD_APPS_ENABLE=testing
+      - OWNCLOUD_LOGLEVEL=0
     commands:
-      - docker run -p 8000:80 owncloud/server:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER}
+      - docker run -p 8000:80 -e DEBUG -e OWNCLOUD_LOGLEVEL -e OWNCLOUD_APPS_INSTALL -e OWNCLOUD_APPS_ENABLE owncloud/server:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER}
 
-  test:
+  prepare-acceptance-tests:
+    image: owncloudci/php
+    pull: true
+    commands:
+      - mkdir -p /drone/owncloud-smoketest
+      - wget -qO- "https://download.owncloud.org/community/owncloud-daily-stable10-qa.tar.bz2" | tar -xj -C /drone/owncloud-smoketest --strip 1
+
+  wait-for-owncloud:
     image: owncloud/ubuntu:latest
     pull: true
     commands:
       - wait-for-it -t 600 docker:8000
-      - curl -sSf http://docker:8000/status.php
+
+#  run-api-smoke-test:
+#     image: owncloudci/php
+#     pull: true
+#     environment:
+#      - TEST_SERVER_URL=http://docker:8000
+#      - SKELETON_DIR=/mnt/data/apps/testing/data/apiSkeleton
+#     commands:
+#      - cd /drone/owncloud-smoketest/tests/acceptance
+#      - bash run.sh --remote --tags @smokeTest --type api
+
+  mailhog:
+    image: toolhippie/docker:latest
+    pull: true
+    detach: true
+    environment:
+      - DOCKER_HOST=tcp://docker:2375
+    commands:
+      - docker run -p 1025:1025 -p 8025:8025 mailhog/mailhog
+
+  run-ui-smoke-test:
+    image: owncloudci/php
+    pull: true
+    environment:
+     - BROWSER=chrome
+     - SELENIUM_HOST=selenium
+     - SELENIUM_PORT=4444
+     - PLATFORM=Linux
+     - MAILHOG_HOST=docker
```

Comment:
I think this needs to be:
```
MAILHOG_HOST=mailhog
```

The name of the mailhog step becomes a hostname in the system pointing to the IP address given to that docker container.

Reply:
That is however not routed from the outside - we are not using a drone service

We are building+starting the owncloud image in a "docker-in-docker" (dind) container. That is self-contained ( similar to a vm). 
So services from drone are not reachable from dind - but we can reach from the outside into dind with the published ports

Reply:
with-in a user-define network 
https://github.com/owncloud-docker/server/pull/77/files#diff-3216dfff0ed3e301453e6799e8c367e2R50

and `docker` alias
https://github.com/owncloud-docker/server/pull/77/files#diff-3216dfff0ed3e301453e6799e8c367e2R83

it works now